File: code/1008/1008.js
const roundFormatted = (number, precision) => {
  const powPrecision = Math.pow(10, precision);
  const roundNumber = (Math.round(number * powPrecision) / powPrecision)
    .toString()
    .split(".");

  return `${roundNumber[0]}.${(roundNumber[1] || "").padEnd(precision, "0")}`;
};

const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const emp = parseInt(lines[0]);
const hours = parseInt(lines[1]);
const salary = parseFloat(lines[2]);

console.log(`NUMBER = ${emp}`);
console.log(`SALARY = U\$ ${roundFormatted(hours * salary, 2)}`)

File: code/1008/1008.cpp
#include <iomanip>
#include <iostream>
using namespace std;

int main() {

  int emp;
  int hours;
  float salary;

  cin >> emp >> hours >> salary;

  cout << "NUMBER = " << emp << endl;
  cout << "SALARY = U$ " << fixed << setprecision(2) << hours * salary << endl;

  return 0;
}
